certificate-authority/store/cqldb/store.go

```diff
@@ -0,0 +1,129 @@
+package cqldb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/plgd-dev/hub/v2/pkg/cqldb"
+	"github.com/plgd-dev/hub/v2/pkg/fsnotify"
+	"github.com/plgd-dev/hub/v2/pkg/log"
+	"github.com/plgd-dev/hub/v2/pkg/security/certManager/client"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// Document
+const (
+	// cqldbdb has all keys in in lowercase
+	idKey         = "id"
+	ownerKey      = "owner"
+	deviceIDKey   = "deviceid"
+	commonNameKey = "commonname"
+	dataKey       = "data"
+)
+
+type Index struct {
+	Name            string
+	PartitionKey    string
+	SecondaryColumn string
+}
+
+// partition key: idKey
+// clustering key: deviceIDKey
+var primaryKey = []string{idKey, ownerKey, commonNameKey}
+
+// Store implements an Store for cqldb.
+type Store struct {
+	client    *cqldb.Client
+	config    *Config
+	logger    log.Logger
+	closeFunc []func()
+}
+
+func New(ctx context.Context, config *Config, fileWatcher *fsnotify.Watcher, logger log.Logger, tracerProvider trace.TracerProvider) (*Store, error) {
+	certManager, err := client.New(config.Embedded.TLS, fileWatcher, logger)
+	if err != nil {
+		return nil, fmt.Errorf("could not create cert manager: %w", err)
+	}
+	cqldbClient, err := cqldb.New(ctx, config.Embedded, certManager.GetTLSConfig(), logger, tracerProvider)
+	if err != nil {
+		return nil, err
```

Comment:
`certManager.Close` is not needed before returning?

same for line 56

Reply:
Fixed